src/measure_theory/special_functions.lean

```diff
@@ -0,0 +1,187 @@
+/-
+Copyright (c) 2019 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Abhimanyu Pallavi Sudhir, Jean Lo, Calle Sönne
```

Comment:
I think that in fact @urkud should be the author of this new file; see #4314, #6041.

Reply:
Fixed, thanks!